docs/learn/fundamentals/contract-development/storage/state-archival.mdx

```diff
@@ -149,6 +153,12 @@ liveUntilLedger that is large enough.
 
 ### RestoreFootprintOp
 
+:::info
+
+This manual operation (`RestoreFootprintOp`) is no longer needed starting in Protocol 23, as archived entries are automatically restored when they appear in a footprint during `InvokeHostFunctionOp`. This section is left here for historical context.
```

Comment:
I think this is a good place to have a sub-section on use-cases for this op that I've listed in another comment.